hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/TransferWithCustomFixedFees.java

```diff
@@ -1592,6 +1594,98 @@ public HapiSpec transferWithInsufficientCustomFee() {
                         .hasKnownStatus(INSUFFICIENT_SENDER_ACCOUNT_BALANCE_FOR_CUSTOM_FEE));
     }
 
+    @HapiTest
+    public HapiSpec transferMultipleTimesWithFixedFeeInHBarShouldVerifyEachTransferIsPaid() {
+        return defaultHapiSpec("transferMultipleTimesWithFixedFeeShouldVerifyEachTransferIsPaid")
+                .given(
+                        cryptoCreate(hbarCollector).balance(ONE_HUNDRED_HBARS),
+                        cryptoCreate(tokenTreasury).balance(ONE_HUNDRED_HBARS),
+                        cryptoCreate(tokenOwner).balance(ONE_MILLION_HBARS),
+                        cryptoCreate(alice).balance(ONE_MILLION_HBARS),
+                        cryptoCreate(bob).balance(ONE_MILLION_HBARS),
+                        cryptoCreate(carol).balance(ONE_MILLION_HBARS),
+                        cryptoCreate(ivan),
+                        tokenCreate(fungibleToken)
+                                .treasury(tokenTreasury)
+                                .initialSupply(tokenTotal)
+                                .payingWith(tokenTreasury)
+                                .withCustom(fixedHbarFee(1L, hbarCollector)),
+                        tokenAssociate(alice, fungibleToken),
+                        tokenAssociate(bob, fungibleToken),
+                        tokenAssociate(carol, fungibleToken),
+                        tokenAssociate(ivan, fungibleToken),
+                        tokenAssociate(tokenOwner, fungibleToken),
+                        cryptoTransfer(moving(tokenTotal, fungibleToken).between(tokenTreasury, tokenOwner)))
+                .when(
+                        cryptoTransfer(moving(10L, fungibleToken).between(tokenOwner, alice))
+                                .fee(1L),
+                        cryptoTransfer(moving(5L, fungibleToken).between(alice, bob))
+                                .fee(1L),
+                        cryptoTransfer(moving(2L, fungibleToken).between(bob, carol))
+                                .fee(1L),
+                        cryptoTransfer(moving(1L, fungibleToken).between(carol, ivan))
+                                .fee(1L))
+                .then(
+                        getAccountBalance(hbarCollector).hasTinyBars(ONE_HUNDRED_HBARS + 4L),
+                        getAccountBalance(alice).hasTokenBalance(fungibleToken, 5L),
+                        getAccountBalance(alice).hasTinyBars(ONE_MILLION_HBARS - 1L),
```

Comment:
you can combine several checks into method chaining i.e: `getAccountBalance(alice).hasTinyBars(ONE_MILLION_HBARS - 1L).hasTokenBalance(fungibleToken, 5L)`

**_apply to all similar occurrences_**

Reply:
@petreze , thanks, I applied the suggested changes. Please re-review.